tests/ecm_prep_test.py

```diff
@@ -66924,6 +66925,22 @@ def setUpClass(cls):
             # Set sector shapes for 3rd example
             if i == 2:
                 cls.opts_emm[i].sect_shapes = True
+        # Options for valid packages test
+        cls.opts_pkgs = copy.deepcopy(opts)
+        cls.opts_pkgs.ecm_files = ["ENERGY STAR Res. ASHP (FS)",
+                                   "Res. Air Sealing (New), IECC c. 2021",
+                                   "Res. Air Sealing (Exist), IECC c. 2021",
+                                   "ENERGY STAR Windows v. 7.0",
+                                   "Residential Walls, IECC c. 2021",
+                                   "Residential Roof, IECC c. 2021",
+                                   "Residential Floors, IECC c. 2021",
+                                   "ENERGY STAR Res. ASHP (FS) CC",
+                                   "ENERGY STAR Res. ASHP (LFL)",
+                                   "ENERGY STAR Res. ASHP (RST)"]
+        cls.opts_pkgs.ecm_packages = ["ENERGY STAR Res. ASHP (FS) + Env.",
+                                      "ENERGY STAR Res. ASHP (FS) + Env. CC",
+                                      "Prosp. Res. ASHP (FS) + Env. + Ctls."]
+        cls.package_ecms_file = fp.ECM_DEF / "package_ecms.json"
```

Comment:
Should we have any concerns about future changes in package_ecms.json or the available ECMs causing this test to fail?

Reply:
The test that uses this does run a higher risk of failing with changes, I copied in package_ecms.json into the new test_files/ecm_definitions directory to be consistent with changes above.